contracts/src/NativeTokenDestination.sol

```diff
@@ -118,61 +120,54 @@ contract NativeTokenDestination is
     uint256 public lastestBurnedFeesReported;
 
     /**
-     * @notice The wrapped native token contract that represents the native tokens on this chain.
+     * @dev When modifier is used, the function can only be called once the contract is fully collelateralized,
+     * account for the initialReserveImbalance.
```

Comment:
```suggestion
     * to account for the initialReserveImbalance.
```

Reply:
Actually meant this to as "accounting for". Thanks for catching 🙏